packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,126 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 5px);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-small-horizontal-inset, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
```

Comment:
remove this background-color from `.notification-dialog-body-text` since already applied in the `.notification-dialog-body` which is the correct place.